frontend/packages/dev-console/integration-tests/tests/serverless.scenario.ts

```diff
@@ -0,0 +1,134 @@
+    await browser.wait(until.elementToBeClickable(submitCount));
+    await submitCount.click();
+    expect(true).toBeTruthy();
+  });
+
+  it('displays subscription creation form for selected Operator', async () => {
+    await crudView.isLoaded();
+    await browser.wait(until.presenceOf(sidenavView.navSectionFor('Operators')));
+    await sidenavView.clickNavLink(['Operators', 'OperatorHub']);
+    await crudView.isLoaded();
+    await catalogView.categoryTabsPresent();
+    await catalogView.categoryTabs.get(0).click();
+    await catalogPageView.catalogTileFor('OpenShift Serverless Operator').click();
+    await browser.wait(until.visibilityOf(installServerlessOperator));
+    await installServerlessOperator.element(by.linkText('Install')).click();
+    await operatorHubView.createSubscriptionFormLoaded();
+    expect(operatorHubView.createSubscriptionFormName.getText()).toEqual(
+      'OpenShift Serverless Operator',
+    );
+    await browser.wait(until.visibilityOf(operatorHubView.createSubscriptionFormInstallMode));
+    await operatorHubView.allNamespacesInstallMode.click();
+
+    expect(operatorHubView.createSubscriptionError.isPresent()).toBe(false);
+    expect(operatorHubView.createSubscriptionFormBtn.getAttribute('disabled')).toEqual(null);
+    await operatorHubView.createSubscriptionFormBtn.click();
+    await crudView.isLoaded();
+    await browser.get(`${appHost}/operatorhub/ns/${testName}`);
+    await crudView.isLoaded();
+    await catalogPageView.clickFilterCheckbox('installState-installed');
+    // Extend the default jasmine timeout interval just in case it takes a while for the htpasswd idp to be ready
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_EXTENDED_TIMEOUT_INTERVAL;
+    expect(catalogPageView.catalogTileFor('OpenShift Serverless Operator').isDisplayed()).toBe(
+      true,
+    );
+  });
+
+  it(`displays Operator in "Cluster Service Versions" view for "${testName}" namespace`, async () => {
+    await retry(() => catalogPageView.catalogTileFor('OpenShift Serverless Operator').click());
+    await browser
+      .wait(until.presenceOf(installServerlessOperator), 1000)
+      .then(() => browser.sleep(500));
+    await operatorHubView.viewInstalledOperator();
+    await crudView.isLoaded();
+    await browser.wait(
+      until.visibilityOf(operatorView.rowForOperator('OpenShift Serverless Operator')),
+    );
+    // Extend the default jasmine timeout interval just in case it takes a while for the htpasswd idp to be ready
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_EXTENDED_TIMEOUT_INTERVAL;
+    expect(operatorView.rowForOperator('OpenShift Serverless Operator').isDisplayed()).toBe(true);
+  });
+
+  it(`create knative-serving namespace and apply serving.yaml`, async () => {
+    await execSync(`oc new-project knative-serving`);
+    await execSync(
+      `oc apply -f ./packages/dev-console/integration-tests/views/serverless-service.yaml`,
+    );
+    expect(true).toBeTruthy();
```

Comment:
same as above , won't it be always true?

Reply:
We have to end the it block with an expect statement otherwise it won't go to other it block. These are the execution of the oc commands, that's why I have used this so that it will go to next it block

Reply:
it doesn't make sense to have it block which will never fail